Scheduling_Algorithm/C++/Preemptive/Round_Robin_Scheduling.cpp

```diff
@@ -0,0 +1,101 @@
+//Round Robin Scheduling is a CPU scheduling algorithm where each process is assigned a fixed time slot in a cyclic way.
```

Comment:
This single line comment style does not look neat. Please add the entire thing inside a multi line comment.